DeAceroDLabs/marketplace-frontend
# [DMP-748] Remove items from offers on Offers and Payment

## What is done

[[DMP-748] [Frontend] Remover Productos de Oferta Seleccionada](https://deacero.atlassian.net/browse/DMP-748)

## How is done

1. Added a handleRemove function recieving a ArticileDetail param
2. Call handleRemove depending on clicked delete item
3. Use setters for context to update offer
4. Couldn't figure out why the component wasn't rerendering when an item was deleted, so added a random seed to force an update (Refactor Pending)

## How to test

1. Run the project via `npm start`
2. Add several products to cart (3 is more than enough)
3. Head over to cart
4. Select an offer
5. Try to remove an item
6. Item should disappear and price should update
7. Head over to payment
8. Try to remove an item
9. Item should disappear and price should update

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
